src/Evolution/Systems/RadiationTransport/M1Grey/Tags.hpp

```diff
@@ -31,12 +31,34 @@ struct TildeE : db::SimpleTag {
 };
 
 /// The densitized momentum density of neutrinos of a given species
-/// \f${\tilde F_i}\f$
+/// \f${\tilde S_i}\f$
 template <typename Fr, class Species>
-struct TildeF : db::SimpleTag {
+struct TildeS : db::SimpleTag {
   using type = tnsr::i<DataVector, 3, Fr>;
   static std::string name() noexcept {
-    return Frame::prefix<Fr>() + "TildeF_" + neutrinos::get_name(Species{});
+    return Frame::prefix<Fr>() + "TildeS_" + neutrinos::get_name(Species{});
+  }
+};
+
+/// The densitized pressure tensor of neutrinos of a given species
+/// \f${\tilde P^{ij}}\f$
+/// computed from \f${\tilde E}\f$, \f${\tilde S_i}\f$ using the M1 closure
+template <typename Fr, class Species>
+struct TildeP : db::SimpleTag {
+  using type = tnsr::II<DataVector, 3, Fr>;
+  static std::string name() noexcept {
+    return Frame::prefix<Fr>() + "TildeP_" + neutrinos::get_name(Species{});
+  }
+};
+
+/// The upper index momentum density of a neutrino species.
+/// This tag does not know the species of neutrinos being used.
+/// \f${\tilde S^i}\f$
```

Comment:
Not a change request:

For my own edification: why doesn't it need to know? (feel free to add it to the dox comment if it shouldn't be obvious) If it's only used inside the cpp file in a temp buffer my vote is to move it into the cpp file in an anonymous namespace

Reply:
It is only used in the cpp file, where i initially had it in an unnamed namespace.
@carmaza requested that i moved the tag to the neutrino namespace, as other functions may use it in the future.
I don't have a strong opinion either way, just let me know what's the agreed upon convention.